client-api/src/main/java/org/corfudb/runtime/kv/core/DbValue.java

```diff
@@ -0,0 +1,107 @@
+package org.corfudb.runtime.kv.core;
```

Comment:
If we put everything into one package it could be too big. If we do it right now it will be:
DbKey,
DbValue,
DbVersion,
Crud,
Record,
Result,
OperationResult and so on

In my opinion, it's too much